--- a/src/schema.rs
+++ b/src/schema.rs
@@ -1,20 +1,26 @@
 // @generated automatically by Diesel CLI.
 
 diesel::table! {
+    use diesel::sql_types::{Nullable, Text};
+    use crate::models::integration::IntegrationTypeMapping;
+
     integrations (id) {
         id -> Text,
         name -> Text,
-        integration_type -> Text,
+        integration_type -> IntegrationTypeMapping,
         url -> Nullable<Text>,
         api_key -> Nullable<Text>,
     }
 }
 
 diesel::table! {
+    use diesel::sql_types::{Nullable, Text, Timestamp};
+    use crate::models::job::{JobStatusMapping, JobTypeMapping};
+
     jobs (id) {
         id -> Text,
-        job_type -> Text,
-        status -> Text,
+        job_type -> JobTypeMapping,
+        status -> JobStatusMapping,
         integration_id -> Nullable<Text>,
         error -> Nullable<Text>,
         created_at -> Timestamp,
@@ -24,13 +30,16 @@
 }
 
 diesel::table! {
+    use diesel::sql_types::{Nullable, Text};
+    use crate::models::library::LibraryTypeMapping;
+
     libraries (id) {
         id -> Text,
         name -> Text,
         external_id -> Nullable<Text>,
         folder -> Nullable<Text>,
         cover -> Nullable<Text>,
-        kind -> Text,
+        kind -> LibraryTypeMapping,
         integration_id -> Nullable<Text>,
     }
 }
